docsets/WPILib_Java/docset.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "WPILib_Java",
+    "version": "2020.2.2",
+    "archive": "WPILib_Java.tgz",
```

Comment:
If possible, please set the docset's name (the `CFBundleName` within `Info.plist`) to "WPILib Java"

Reply:
I'll do the same for the C++ version as well (once I actually include it lol)

Reply:
Should I touch the other keys as well?  Eg, DocSetFeedName or Publisher?

Reply:
Those keys aren't used by Dash. `DocSetPlatformFamily` and `CFBundleName` are the important ones for Dash.